src/composables/accountSelector.ts

```diff
@@ -0,0 +1,195 @@
+import {
+  computed,
+  ref,
+  watch,
+} from 'vue';
+
+import type {
+  IAccountSelectorEntry,
+  ITransaction,
+} from '@/types';
+import {
+  ACCOUNT_SELECT_TYPE_FILTER,
+  AccountSelectTypeFilter,
+  PROTOCOLS,
+  TX_DIRECTION,
+} from '@/constants';
+import {
+  createCustomScopedComposable,
+  excludeFalsy,
+  getDefaultAccountLabel,
+  getDirection,
+} from '@/utils';
+import {
+  useAccounts,
+  useAddressBook,
+  useLatestTransactionList,
+} from '@/composables';
+
+import { useAeNames } from '@/protocols/aeternity/composables/aeNames';
+import { getInnerTransaction, getOwnershipStatus, getTxOwnerAddress } from '@/protocols/aeternity/helpers';
+
+import { tg } from '@/popup/plugins/i18n';
+
+export const useAccountSelector = createCustomScopedComposable(() => {
+  const { getName } = useAeNames();
+
+  const searchQuery = ref('');
+  const {
+    addressBookFiltered: rawAddressBookFiltered,
+    addressBookFilteredByProtocol,
+    protocolFilter,
+    showBookmarked,
+    getAddressBookEntryByAddress,
+    setProtocolFilter,
+    setShowBookmarked,
+    clearFilters: addressBookClearFilters,
+  } = useAddressBook();
+  const {
+    activeAccount,
+    accounts,
+    accountsGroupedByProtocol,
+    getAccountByAddress,
+  } = useAccounts();
+  const { accountsTransactionsLatest } = useLatestTransactionList();
+  const accountSelectType = ref<AccountSelectTypeFilter>(ACCOUNT_SELECT_TYPE_FILTER.addressBook);
+  const addressBookFiltered = computed(() => rawAddressBookFiltered.value.map((account) => ({
```

Comment:
In most cases we have this unwritten rule to make one empty line between multiline computed properties.